src/Foundation/Base/code/ModelMappers/IMapper.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Wooli.Foundation.Base.ModelMappers
```

Comment:
- Please move to "Mappers" folder 
- Also, please update Guidelines with description of these changes and how to use it

Reply:
I'll update guidelines after merge